pkg/plugins/scms/github/scm.go

```diff
@@ -21,11 +23,13 @@ func (g *Github) Clean() error {
 
 // Clone run `git clone`.
 func (g *Github) Clone() (string, error) {
-
-	URL := fmt.Sprintf("https://github.com/%v/%v.git",
+	URL := fmt.Sprintf("%v/%v/%v.git",
+		g.Spec.URL,
 		g.Spec.Owner,
 		g.Spec.Repository)
 
+	logrus.Debug("Cloning repository", URL)
```

Comment:
This debug message is already displayed in the the gitgeneric library
```suggestion
```

So if you had the parameter --debug to your command you'll see the message